.vscode/settings.json

```diff
@@ -46,5 +46,97 @@
   "typescript.tsdk": "node_modules\\typescript\\lib",
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
+  },
+  "files.associations": {
```

Comment:
doublechecking if this is intentional.....

Reply:
Hmm wasn't intentional not sure what I did that added this. I'll remove it now.